src/index.css

```diff
@@ -5,6 +5,13 @@
 html {
   background-color: #0d0030;
   overflow-x: hidden;
+  /* Grid */
```

Comment:
I've noticed, that grid doesn't apply for `footer` if we use grid inside the body